tests/test_ami.py

```diff
@@ -100,22 +100,49 @@ def test_el7_fips_disabled(host):
     assert not fips.exists or fips.content.strip() == b'0'
 
 
-@pytest.mark.parametrize("name", [
-    ("python36")
+@pytest.mark.parametrize("names", [
```

Comment:
So, pytest effectively treats this as an ORed test?

Reply:
The logic in the function does, yes. We are just telling pytest to pass a specific list of packages to the function. The function loops over the items in the list, breaks if the package is installed, and then asserts the installation status. If none of the pkgs are installed, the assert fails and pytest fails the test.

Reply:
woot